config/initializers/airbrake.rb

```diff
@@ -1,8 +1,10 @@
 if ENV['ERRBIT_API_KEY'].present?
+  errbit_uri = Plek.find('errbit')
+
   Airbrake.configure do |config|
-    config.api_key = ENV['ERRBIT_API_KEY']
-    config.host = "errbit.#{ENV['GOVUK_APP_DOMAIN']}"
-    config.secure = true
-    config.environment_name = ENV['ERRBIT_ENVIRONMENT_NAME']
+    config.project_key = ENV['ERRBIT_API_KEY']
+    config.project_id = 1 # dummy, not used in Errbit
+    config.host = errbit_uri
```

Comment:
Just found this page, and it shows in examples that the host doesn't have to be a hostname, so maybe this is not an issue https://github.com/airbrake/airbrake/blob/master/docs/Migration_guide_from_v4_to_v5.md#port .